### ISSUE TITLE
when I update vue version 2.4.3,  both text and pictures can not be displayed in my weex project

### ISSUE THREAD

**Issue by hacke2** [vuejs/vue#6598]
### Version
2.4.3

### Reproduction link
[https://github.com/hacke2/weex-vue-2.4.3-bug](https://github.com/hacke2/weex-vue-2.4.3-bug)

### Steps to reproduce
npm install
npm run dev

### What is expected?
see text and picture

### What is actually happening?
dom does not display properly

---
when I use vue 2.4.2, that is ok

**Comment by MrRaindrop** [comment]
weex's `text` and `image` component has been recognize as a svg element and has been set as in the 'svg' namespace. So I've hacked the `Vue.config.getTagNamespace` to ignore all the weex's components. And it works fine afterwards.

https://github.com/MrRaindrop/incubator-weex/commit/2cf16754205ec74fd4e646fecf36db84ced5b588#diff-1da999564da5a4bfd3be47a747d5bd6bR52

[v0.12.12](https://www.npmjs.com/package/weex-vue-render) has been released to fix this problem. Pls to check it out.
